CorsaSchoolManagementSystem/Views/addStudent.php

```diff
@@ -7,15 +7,16 @@
     <title>Corsa And Partners Boarding School Complex</title>
     <link rel="icon" href="<?php echo BASE_URL?>/Views/image/logo.jpg">
     <link rel="stylesheet" href="https://bootswatch.com/5/cosmo/bootstrap.min.css">
-    <!-- CSS only -->
+    <!-- Bootstrap Icon -->
```

Comment:
Why bootstrap Icons? 
I thought we agreed on using one standard throughout. If we started with Material Icons, we should maintain that through out.